pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -130,7 +130,7 @@ const (
 	DefaultFluentbitImageRepository             = "fluent/fluent-bit"
 	DefaultFluentbitImageTag                    = "2.1.4"
 	DefaultFluentbitBufferVolumeImageRepository = "ghcr.io/kube-logging/node-exporter"
-	DefaultFluentbitBufferVolumeImageTag        = "v0.6.1"
+	DefaultFluentbitBufferVolumeImageTag        = "v0.7.0"
```

Comment:
can you give us a snapshot for the old vs new metrics endpoints results so that we can see the difference?

Reply:
Added to OP. For some reason HELP text is no more present when exporting 3 files. I could not find why.